data/scannet/README.md

```diff
@@ -11,7 +11,7 @@ We follow the procedure in [votenet](https://github.com/facebookresearch/votenet
 4. Enter the project root directory, generate training data by running
 
 ```bash
-python tools/create_data.py scannet --root-path ./data/scannet --out-dir ./data/scannet --extra-tag scannet
+python tools/create_data.py --dataset scannet --root-path ./data/scannet --out-dir ./data/scannet --extra-tag scannet
```

Comment:
It seems we do not need to add `--dataset` argument.

Reply:
I am not sure how you run this script, but on my end, when I run the script without the `--dataset` argument ( for scannet for example), I get the following error:


```
usage: create_data.py [-h] [--dataset kitti] [--root-path ROOT_PATH] [--version VERSION] [--max-sweeps MAX_SWEEPS] --out-dir OUT_DIR [--extra-tag EXTRA_TAG]
                      [--workers WORKERS]
create_data.py: error: unrecognized arguments: scannet
```

Reply:
I believe this [line](https://github.com/open-mmlab/mmdetection3d/blob/main/tools/create_data.py#L269) makes the script expecting the `--dataset` argument.

Reply:
I am using python 3.10 btw.